test/e2e/annotation/annotation_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Authors of spidernet-io
+// SPDX-License-Identifier: Apache-2.0
+package annotation_test
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spidernet-io/e2eframework/tools"
+	"github.com/spidernet-io/spiderpool/test/e2e/common"
+)
+
+var _ = Describe("test annotation", Label("annotation"), func() {
+	var nsName, podName string
+
+	BeforeEach(func() {
+		// init namespace name and create
+		nsName = "ns" + tools.RandomName()
+		GinkgoWriter.Printf("create namespace %v \n", nsName)
+		err := frame.CreateNamespace(nsName)
+		Expect(err).NotTo(HaveOccurred(), "failed to create namespace %v", nsName)
+
+		// init test name
+		podName = "pod" + tools.RandomName()
+
+		// clean test env
+		DeferCleanup(func() {
+			GinkgoWriter.Printf("delete namespace %v \n", nsName)
+			err = frame.DeleteNamespace(nsName)
+			Expect(err).NotTo(HaveOccurred(), "failed to delete namespace %v", nsName)
+		})
+	})
+
+	DescribeTable("invalid annotations table", func(keyName common.AnnotationKey, keyValue string) {
+		// try to create pod
+		GinkgoWriter.Printf("try to create pod %v/%v with annotation key/value=\n %v: \n %v \n", nsName, podName, keyName, keyValue)
```

Comment:
try to create pod %v/%v with annotation key/value=\n %v: \n %v \n
---》
try to create pod %v/%v with annotation %v=%v  \n